docs/release/3.4/release-3-4-23.md

```diff
@@ -0,0 +1,224 @@
+OSG Software Release 3.4.23
+===========================
+
+**Release Date**: 2019-01-23
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   Gratia probes 1.20.8
+    -   Interpret CPU expressions for Hosted CEs (Needed for proper accounting)
+    -   Set Processors field properly for Slurm
+    -   Tests: validate processors field
+-   Upcoming Repository
+    -   [Singularity 3.0.2](https://github.com/sylabs/singularity/releases/tag/v3.0.2)
+        -   Reimplemented in Go
+        -   [Many changes](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md)
+    -   [HTCondor 8.8.0](https://www-auth.cs.wisc.edu/lists/htcondor-world/2019/msg00000.shtml)
+        -   Job Router: follow first match by default, easier to write routing rules
```

Comment:
I think we need to make this a `!!! danger` or something, indicate that this is a change in behavior that may effect sites with multiple routes. Then provide the configuration value to set for the old value and how to apply that configuration.